modules/android/_preload.lua

```diff
@@ -19,6 +19,15 @@
 	api.addAllowed("flags", { "Thumb" })
 	api.addAllowed("kind", p.PACKAGING)
 
+	api.addAllowed("toolchainversion", {
```

Comment:
Just wondering why we moved the order of this?

Reply:
I wanted to put them next to the other addAllowed functions, since it has changed from using the register function. Now the main preload registers the api, and both linux and android add their allowed versions